kornia/feature/defmo.py

```diff
@@ -0,0 +1,112 @@
+from typing import Dict
+
+import torch
+import torch.nn as nn
+import torchvision.models
```

Comment:
That breaks basic kornia reliance on the torch-only. Could you please add (copy-paste) some definition of the relevant blocks to the defmo? I understand, that it is a bit more cumbersome, but being a dependence-less, is an improtant feature.

Reply:
+1 add whatever code network is proven that works well + proper reference from where it’s taken

Reply:
Done.